### PR TITLE
I would like to add this project. Thank You.

### PR DESCRIPTION
**Pull request by blavejr**

### CODE REVIEW

File: _data/projects/OrganiseDesktop.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/blavejr/OrganiseDesktop
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- Python
+- tkinter
+- GUI
+- easy
+upforgrabs:
+  name: help needed
+  link: https://github.com/blavejr/OrganiseDesktop/issues
```
Comment (shiftkey) [inline _data/projects/OrganiseDesktop.yml]: This should be a link to the issues labelled with `help wanted`, e.g. `https://github.com/blavejr/OrganiseDesktop/labels/help%20wanted`

**Comment by shiftkey** [comment]
Looks good aside from that one comment, but I'm going to hold off on merging until there are some issues on the repository which are labelled for new contributors. Please let me know when that's done and I can merge this in!

**Comment by blavejr** [comment]
Hi @shiftkey thank you for the reply, I have made the necessary changes.

**Comment by shiftkey** [comment]
@blavejr thanks!

**Pull request merged**